sci-geosciences/sfcgal/sfcgal-1.3.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+MY_P=SFCGAL-${PV}
+
+DESCRIPTION="C++ library for geometric algorithms and data structures"
+HOMEPAGE="http://www.sfcgal.org/"
+SRC_URI="https://github.com/Oslandia/SFCGAL/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Have you actually tested the Prefix install?

Reply:
I tested it in x86 chroot env